crates/joystick/src/windows.rs

```diff
@@ -81,88 +52,79 @@ impl Joystick {
         Joystick::default()
     }
 
-    pub fn name(&self) -> &str {
-        &self.name
-    }
-
     pub fn tap_a(&mut self) {
-        self.tap(KeyCode::BTN_EAST)
+        self.tap(XButtons::A)
     }
 
     pub fn tap_b(&mut self) {
-        self.tap(KeyCode::BTN_SOUTH)
+        self.tap(XButtons::B)
     }
 
     pub fn tap_x(&mut self) {
-        self.tap(KeyCode::BTN_NORTH)
+        self.tap(XButtons::X)
     }
 
     pub fn tap_y(&mut self) {
-        self.tap(KeyCode::BTN_WEST)
+        self.tap(XButtons::Y)
     }
 
     pub fn tap_lt(&mut self) {
-        self.tap(KeyCode::BTN_TL)
+        self.tap(XButtons::LB)
     }
 
     pub fn tap_rt(&mut self) {
-        self.tap(KeyCode::BTN_TR)
+        self.tap(XButtons::RB)
     }
 
+    // TODO(eein): We dont rear triggers yet
```

Comment:
Any reason to not have them active here though? Do they not work? This is general code, not game specific.

Reply:
Was mostly just implementing what i had on the linux side - will make a note in the tracking issue to add these. (linux has them configured, just need to add the api)